docs/pipeline/metadata.md

```diff
@@ -75,11 +97,15 @@ This activity typically happens at the end of year or, start of the year.
 1. Add a new sheet to Lab Metadata Tracking Google spreadsheet
 2. Create Portal PR to make it aware. REF:
     - https://github.com/umccr/data-portal-apis/pulls?q=is%3Apr+is%3Aclosed+lab+metadata+sheet
-    - https://github.com/umccr/data-portal-apis/pull/535/files
-    - https://github.com/umccr/data-portal-apis/pull/653/files
+    - https://github.com/umccr/data-portal-apis/pull/535
+    - https://github.com/umccr/data-portal-apis/pull/653
 3. Create Infrastructure PR to update SubjectID generator App script
     - https://github.com/umccr/infrastructure/blob/master/scripts/umccr_pipeline/UMCCR_Library_Tracking_MetaData.js#L5
     - https://github.com/umccr/infrastructure/pull/344
+    - https://github.com/umccr/infrastructure/pull/391
 4. Deploy SubjectID generator App script
+5. Update SampleSheet Checker UI (if any)
+   - https://github.com/umccr/samplesheet-check-frontend/pull/40
+   - https://umccr.slack.com/archives/C94CQMNVA/p1704844116202929
```

Comment:
@williamputraintan Added sschecker here too. And future of Metadata Manager; keep an eye on  below trello card too; we shall drive from there. fyi